app/views/user_orders/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Profile Orders</h1>
+
+<section class="orders-index">
+  <% @user.orders.each do |order| %>
+    <p><%= link_to "#{order.id}", "/orders/#{order.id}" %></p>
+    <p><%= order.name %></p>
+    <p><%= order.address %></p>
+    <p><%= order.status %></p>
+    <p><%= order.created_at %></p>
```

Comment:
When we get to the user story that deals with orders index, we should add some context to this variables